### PR TITLE
Add Player.Lift property + Elevator API

### PR DESCRIPTION
**Pull request by Thundermaker300**

### CODE REVIEW

**Review by iopietro — APPROVED** [review]

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -873,6 +873,23 @@ public string GroupName
...
         public Room CurrentRoom => Map.FindParentRoom(GameObject);
 
+        /// <summary>
+        /// Gets the current <see cref="API.Features.Lift"/> the player is in. Can be <see langword="null"/>.
+        /// </summary>
+        public Lift Lift
+        {
+            get
+            {
+                foreach (var lift in Lift.List)
```
Comment (louis1706) [inline Exiled.API/Features/Player.cs:883]: idk if it's possible but we have possibly a way to make it better

File: Exiled.API/Features/Lift.cs [hunk 2]
```diff
@@ -49,6 +49,14 @@ internal Lift(ElevatorChamber elevator)
                 internalDoorsList.Add(door);
         }
 
+        /// <summary>
+        /// Finalizes an instance of the <see cref="Lift"/> class.
+        /// </summary>
+        ~Lift()
+        {
+            ListPool<ElevatorDoor>.Pool.Return(internalDoorsList);
+        }
```
Comment (iopietro) [inline Exiled.API/Features/Lift.cs:58]:
```suggestion
        ~Lift() => ListPool<ElevatorDoor>.Pool.Return(internalDoorsList);
```

**Review by iopietro — CHANGES_REQUESTED** [review]

**Comment by louis1706** [comment]
https://github.com/Thundermaker300/EXILED/pull/3 suggested change

**Comment by Thundermaker300** [comment]
Will recreate without a breaking pr from yamato :trollface:

**Pull request closed without merging**